test/unit/support/loadCertfile.test.js

```diff
@@ -0,0 +1,22 @@
+/* global describe,it,process */
+const expect = require('chai').expect;
+const loadCertFile = require('../../../lib/support/loadCertFile.js');
+const https = require('https');
+const path = require('path');
+
+describe('Cert file Loder', () => {
+  it('load the SSL certification file when a environment variable is avaiable', () => {
+    process.env.SSL_CERT_FILE = path.join(__dirname, 'certfile.test.cert');
+    expect(https.globalAgent.options.ca).haveLengthOf(0);
+    loadCertFile();
+    expect(https.globalAgent.options.ca).haveLengthOf(1);
+  });
+
+  it('load the SSL certification file when a environment variable is avaiable', () => {
```

Comment:
This message should be something like:  
`should not load SSL certification file if no environment variable is available`